### PR TITLE
fixes set title and handlerUrl on widget init config

### PR DESCRIPTION
**Pull request by Quexer69**
```
echo \hrzg\filemanager\widgets\FileManagerWidget::widget(
   [
        'title' => getenv('APP_TITLE'),
        'handlerUrl' => '/my/handler/url',
   ]
);
```
before, `title` and `handlerUrl` property didnt get recognized in `BaseFilemanagerWidget::setFileManagerConfig()` because the properties were defined in the parent widget class!

I think splitting the `FilemanagerWidget` in a base class is not necessary or gives no advantages so I removed the base class.

@schmunk42 @handcode

### CODE REVIEW

File: src/widgets/FileManagerWidget.php [hunk 1]
```diff
@@ -30,18 +32,118 @@ class FileManagerWidget extends BaseFileManagerWidget
...
+            $this->handlerUrl = Url::to([getenv('AFM_HANDLER_URL')]);
+        }
+
+        // check if user is allowed to change permissions
+        $allowPermissions = false;
+        if (!\Yii::$app->user->isGuest && \Yii::$app->user->can('FileflyPermissions')) {
+            $allowPermissions = true;
+        }
+
+        $title             = empty($this->title) ? getenv('AFM_TITLE') : $this->title;
```
Comment (schmunk42) [inline src/widgets/FileManagerWidget.php:78]: This line is also changed compared to the original version, something else we need to look at?

**Review by schmunk42 — COMMENTED** [review]
In general it's OK for a widget not to have a base-class, but the changes would be much easier to grasp if you would have added just the missing properties to the base class. @handcode WDYT?

File: src/widgets/FileManagerWidget.php (hunk 1 above)
Comment (Quexer69) [reply src/widgets/FileManagerWidget.php:78]: Yes you‘re right schmunk I better should have splitted the MR into two commits... scusi!

No, besides merging the base class and init the title property with an empty string its the only line.

**Comment by schmunk42** [comment]
Thank you!

**Pull request merged**